installation/routines/setup_rfid_reader.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 _run_setup_rfid_reader() {
-    python "${INSTALLATION_PATH}/src/jukebox/run_register_rfid_reader.py" | tee /dev/fd/3
+    python "${INSTALLATION_PATH}/src/jukebox/run_register_rfid_reader.py" | log
```

Comment:
Old behavior was `print_lc`. 

But this code is not working and breaks installation (nothing shown to the user but input is required).
The new logging functions don't work with pipes

Reply:
Weird it worked for me when I tried, but indeed I see it doesn't work for install. Will check this again.